packages/model-viewer/src/features/ar.ts

```diff
@@ -98,12 +98,12 @@ export type QuickLookBrowser = 'safari'|'chrome';
 const deserializeQuickLookBrowsers =
     enumerationDeserializer<QuickLookBrowser>(['safari', 'chrome']);
 
-export type ARMode = 'quick-look'|'scene-viewer'|'webxr'|'fallback'|'none';
+export type ARMode = 'quick-look'|'scene-viewer'|'webxr'|'none';
 
 const deserializeARModes = enumerationDeserializer<ARMode>(
-    ['quick-look', 'scene-viewer', 'webxr', 'fallback', 'none']);
```

Comment:
Drive-by question: if I'm understanding it right, this is removing 'fallback' from the list of recognized values. If a site is still using this in an attribute, will it be ignored silently, or does it cause an error?

Reply:
It will be ignored silently